### PR TITLE
[RW-6502][risk=no]  Create prepackaged (fake) all whole genome variant data concept set

### PR DESCRIPTION
**Pull request by NehaBroad**
WHOLE GENOME concept set will be displayed only if the cdrversion.hasWgsData is false

Selecting the whole genome will display the value 'VCF Files(s)'
Screen Shot 2021-04-07 at 2 42 23 PM

Preview table will display the tab 'VCF File(s)' : Note the table will not display anything.
Screen Shot 2021-04-07 at 2 42 28 PM

---
**PR checklist**

- [x] This PR meets the Acceptance Criteria in the JIRA story
- [x] The JIRA story has been moved to Dev Review
- [x] This PR includes appropriate unit tests
- [x] I have run and tested this change locally
- [ ] I have run the E2E tests on ths change against my local UI and/or API server with `yarn test-local` or [yarn test-local-devup](https://github.com/all-of-us/workbench/blob/master/e2e/README.md#examples)
- [ ] If this includes a UI change, I have taken screen recordings or screenshots of the new behavior and notified the PO and UX designer
- [ ] If this includes an API change, I have updated the appropriate Swagger definitions and notified API consumers
- [ ] If this includes a new feature flag, I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later

### CODE REVIEW

**Review by dolbeew — APPROVED** [review]
LGTM

File: api/src/main/java/org/pmiops/workbench/dataset/DataSetServiceImpl.java [hunk 1]
```diff
@@ -93,6 +93,15 @@
           Domain.PROCEDURE,
           Domain.PHYSICAL_MEASUREMENT_CSS);
 
+  private static final ImmutableList<Domain> DOMAIN_NO_CONCEPT_SET =
```
Comment (als364) [inline api/src/main/java/org/pmiops/workbench/dataset/DataSetServiceImpl.java:96]: nit: would name this something like `DOMAINS_WITHOUT_CONCEPT_SETS`

File: ui/src/app/pages/data/data-set/dataset-page.tsx [hunk 2]
```diff
@@ -625,6 +638,22 @@ const DataSetPage = fp.flow(withUserProfile(), withCurrentWorkspace(), withUrlPa
       return ids.map(id => setsById.get(id));
     }
 
+    getPrePackagedList() {
+      let prepackagedList = Object.keys(PrepackagedConceptSet);
+      if (!getCdrVersion(this.props.workspace, this.props.cdrVersionListResponse).hasFitbitData) {
+        // TO DO Fix spelling mistake in prepackageSet
```
Comment (als364) [inline ui/src/app/pages/data/data-set/dataset-page.tsx:644]: Is there a ticket for this? If so, could you include it in this comment?

File: ui/src/app/pages/data/data-set/dataset-page.tsx [hunk 3]
```diff
@@ -832,6 +865,10 @@ const DataSetPage = fp.flow(withUserProfile(), withCurrentWorkspace(), withUrlPa
           case PrepackagedConceptSet.FITBITHEARTRATELEVEL:
             selectedPrePackagedConceptSDetEnum.push(PrePackagedConceptSetEnum.FITBITHEARTRATELEVEL);
             break;
+          case PrepackagedConceptSet.WHOLEGENOME:
+            selectedPrePackagedConceptSDetEnum.push(PrePackagedConceptSetEnum.WHOLEGENOME);
```
Comment (als364) [inline ui/src/app/pages/data/data-set/dataset-page.tsx:869]: nit: would be nice to fix this at the same time as fixing the FITIBIT typo

File: ui/src/app/utils/index.tsx [hunk 4]
```diff
@@ -438,8 +438,9 @@ export function formatWorkspaceResourceDisplayDate(time: string): string {
 }
 
 export function formatDomainString(domainString: string): string {
-  return fp.capitalize(domainString === Domain.PHYSICALMEASUREMENTCSS.toString()
-    ? Domain.PHYSICALMEASUREMENT.toString() : domainString);
+  return domainString === Domain.PHYSICALMEASUREMENTCSS.toString()
+    ? fp.capitalize(Domain.PHYSICALMEASUREMENT.toString())
+    : domainString === Domain.WHOLEGENOMEVARIANT.toString() ? 'VCF Files' : fp.capitalize(domainString);
```
Comment (als364) [inline ui/src/app/utils/index.tsx:443]: nit: if you move this ?: onto the next two lines, it'll be easier to tell what's going on / how many conditionals there are

**Review by als364 — APPROVED** [review]
A few nits but otherwise lgtm

File: ui/src/app/pages/data/data-set/dataset-page.tsx (hunk 2 above)
Comment (NehaBroad) [reply ui/src/app/pages/data/data-set/dataset-page.tsx]: Fixed in this PR

File: ui/src/app/pages/data/data-set/dataset-page.tsx (hunk 3 above)
Comment (NehaBroad) [reply ui/src/app/pages/data/data-set/dataset-page.tsx:866]: Done

File: ui/src/app/utils/index.tsx (hunk 4 above)
Comment (NehaBroad) [reply ui/src/app/utils/index.tsx]: Done

File: api/src/main/java/org/pmiops/workbench/dataset/DataSetServiceImpl.java (hunk 1 above)
Comment (NehaBroad) [reply api/src/main/java/org/pmiops/workbench/dataset/DataSetServiceImpl.java]: Done

**Pull request merged**